### ISSUE TITLE
react devtools not working for same setup that works for iOS

### ISSUE THREAD

**Issue by faceyspacey** [jhen0409/react-native-debugger#70]
I'm using the same setup I have for iOS, but in Android. The Redux devtools and console works, but for whatever reason the react-devtools don't. I can't find any Android-specific documentation for things I might need to change. 

Do the included React Devtools not work for Android yet?

Here's the error I'm getting:

```
failed to connect
(anonymous) @ Store.js:507
```

```
 _establishConnection() {
    var tries = 0;
    var requestInt;
    this._bridge.once('capabilities', capabilities => {
      clearInterval(requestInt);
      this.capabilities = assign(this.capabilities, capabilities);
      this.emit('connected');
    });
    this._bridge.send('requestCapabilities');
    requestInt = setInterval(() => {
      tries += 1;
      if (tries > 100) {
        console.error('failed to connect'); // line 507!
        clearInterval(requestInt);
        this.emit('connection failed');
        return;
      }
      this._bridge.send('requestCapabilities');
    }, 500);
  }
```
React Native version: 44
Platform: Android
Is real device of platform: NO
Operating System: macOS

**Comment by jhen0409** [comment]
We have a [troubleshoot](https://github.com/jhen0409/react-native-debugger/blob/master/docs/troubleshooting.md#react-inspector-get-stuck-at-connecting-to-react-for-rn-043-45) for this (even have a [PR](https://github.com/facebook/react-native/pull/13116) for react-native waiting for shipped), and it duplicate of #45.

**Comment by faceyspacey** [comment]
oh my bad: the troubleshooting solutions with one change to that code:

let connectToDevTools

  if (__DEV__) {
    // ......
    const bakRIC = window.requestIdleCallback;
    const bakCIC = window.cancelIdleCallback;
    // To ensure react-devtools-core use polyfill instead of requestIdleCallback
    window.requestIdleCallback = null;
    window.cancelIdleCallback = null;
    connectToDevTools = require('react-devtools-core').connectToDevTools; // Line 17
    window.requestIdleCallback = bakRIC;
    window.cancelIdleCallback = bakCIC;
    // ......
  }

i'll make a PR.
